components/server/src/user/user-service.ts

```diff
@@ -163,7 +163,7 @@ export class UserService {
             // blocked = if user already blocked OR is not allowed to pass
             newUser.blocked = newUser.blocked || !canPass;
         }
-        if (!newUser.blocked && (isFirstUser || this.config.makeNewUsersAdmin)) {
+        if (!newUser.blocked && isFirstUser && this.config.admin.grantFirstUserAdminRole) {
```

Comment:
+1

this changes should makes it explicit, that only the first regular signup is upgraded to an admin. 

previously it would be possible to make all signups admin, if the config is set like that. 😨 the use case for that is not clear, my best guess is that it was to unblock preview environments.

Reply:
Jep, I think `makeNewUsersAdmin` is a leftover from the early days. Luckily it was not used anywhere.

IMO we should also deprecate `grantFirstUserAdminRole` (implicit default for now), but that's out of scope here.